core/sagas/blueprints.js

```diff
@@ -182,6 +175,31 @@ function* commitToWorkspace(action) {
   }
 }
 
+function* deleteWorkspace(action) {
```

Comment:
There's a lot of duplication between this function and fetchBlueprintContents, and I couldn't figure out a simple way to handle that without making major changes. Any recommendations are welcome.

Reply:
Yeah, it's duplicated, but hard to entangle right now. I think it's fine to keep it as is for now.